common/protob/messages-benfen.proto

```
@@ -0,0 +1,87 @@
+syntax = "proto2";
+package hw.trezor.messages.benfen;
```

Buf (1.47.2)

Files with package "hw.trezor.messages.benfen" must be within a directory "hw/trezor/messages/benfen" relative to root but were in directory "common/protob".

(PACKAGE_DIRECTORY_MATCH)